tests/test_webhook.py

```diff
@@ -133,15 +139,41 @@ def test_fails_on_not_known_action(self):
 
     def test_extract_browser_label(self):
         """Extract browser label name."""
-        browser_label = helpers.extract_browser_label(self.issue_body)
+        browser_label = helpers.extract_browser_label(
+            helpers.extract_metadata(self.issue_body))
```

Comment:
chaining functions in unittest is usually not a good idea. Just test with the input which is  expected to isolate the tests. So here just dictionaries.

```python
def test_extract_browser_label(self):
    """Extract browser label name."""
    metadata_tests = [
        ({'browser': 'Firefox'}, 'browser-firefox'),
        ({'browser': 'Firefox Mobile'}, 'browser-firefox-mobile'),
        ({'browser': 'Firefox 30.0'}, 'browser-firefox'),
        ({'browser': 'Firefox Mobile 30.0'}, 'browser-firefox-mobile'),
        ({'browser': 'Firefox (tablet)'}, 'browser-firefox-tablet')
        ]
    for metadata_dict, expected in metadata_tests:
        actual = helpers.extract_browser_label(metadata_dict)
        self.assertEqual(expected, actual)
```